docs/content/en/api.md

```diff
@@ -99,6 +99,14 @@ Instance of [VueI18n class](http://kazupon.github.io/vue-i18n/api/#vuei18n-class
 
   Switches locale of the app to specified locale code. If `useCookie` option is enabled, locale cookie will be updated with new value. If prefixes are enabled (`strategy` other than `no_prefix`), will navigate to new locale's route.
 
+#### getBrowserLocale
+
+  - **Arguments**:
+    - no arguments
+  - **Returns**: `string | undefined | null`
+
+  Returns browser locale code filtered against the ones defined in options.
+
```

Comment:
Please also update `es/api.md` in same way.

Reply:
Done thanks to @ferferga!